test/cloud_firestore_mocks_test.dart

```diff
@@ -922,4 +922,32 @@ void main() {
     expect(eve.data['name'], isNot('John')); // nothing changed
     expect(eve.data['friends'], equals(['Alice', 'Bob'])); // nothing changed
   });
+
+  test('CollectionGroup getDocuments', () async {
+    final firestore = MockFirestoreInstance();
+    await firestore.document('foo/foo_1/bar/bar_1').setData({'value': '1'});
+    await firestore.document('foo/foo_2/bar/bar_2').setData({'value': '2'});
+    await firestore.document('bar/bar_3').setData({'value': '3'});
+    final querySnapshot = await firestore.collectionGroup('bar').getDocuments();
+    expect(querySnapshot.documents, hasLength(3));
+    expect(querySnapshot.documents.first.documentID, 'bar_1');
+    expect(querySnapshot.documents.first.reference.path, 'foo/foo_1/bar/bar_1');
```

Comment:
this t DocumentReference also workds by https://github.com/atn832/cloud_firestore_mocks/pull/123/files#diff-c28c8f90e4dd6b8374977f1e7e4d943cR73